interface_frontend.py

```diff
@@ -272,8 +272,14 @@ def draw_interface(interface_state, game_state, window):
         power_down_sprite.draw()
 
     # Timer
-    if interface_state.timer:
-        timer_sprite.draw()
+    if interface_state.timer is not None:
+        seconds = monotonic() - interface_state.timer
+        seconds_left = round(30-seconds)
+
+        timer_label = get_label(f"00:{seconds_left}", 585, 865, 26, "center", (255, 0, 0, 255))
```

Comment:
Co je to za čísla? Pojmenuj je, prosím., ať se v tom dá vyznat. Třeba:
```suggestion
        timer_label = get_label(f"00:{seconds_left}", x=585, y=865, font_size=26, anchor_x="center", color=(255, 0, 0, 255))
```

Reply:
Je to přece pojmenované ve funkci get_label, v celém souboru jsou takto tvořeny. To teda změním u všeho.